### PR TITLE
Cu 86b11czjh profile  badges UI

### PR DESCRIPTION
**Pull request by ReinhardtBotha**
Currently using dummy data to show the list of badges, will create a new ticket for the actual badges.

### CODE REVIEW

**Comment by zeckkimon** [comment]
Task linked: [CU-86b11czjh Profile / Badges UI](https://app.clickup.com/t/86b11czjh)

**Review by whamester — APPROVED** [review]

**Comment by whamester** [comment]
Ticket created to address the TODO comments: https://app.clickup.com/t/86b14hg72

**Review by Victorportus — APPROVED** [review]

File: src/app/badges.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    <ScrollView
+      style={styles.container}
+      contentContainerStyle={styles.scrollViewContent}
+    >
+      <View style={styles.background} />
+      <View style={styles.innerContainer}>
+        <Spacer height={64} />
+
+        <Stack flexDirection="row" alignItems="center">
+          <Pressable onPress={back} style={styles.icon}>
```
Comment (whamester) [inline src/app/badges.tsx]: onPress={router.back}

File: src/components/profile/DailyGoalCard.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
+import { useUser } from "@src/state/useUser";
+import Icon from "../ui/Icon";
+import Stack from "../ui/Stack";
+import { Text } from "../ui/typography";
```
Comment (whamester) [inline src/components/profile/DailyGoalCard.tsx:4]: import relative path


"@src/ui/typography"

File: src/components/profile/StreakCard.tsx [hunk 3]
```diff
@@ -0,0 +1,45 @@
+import { useUser } from "@src/state/useUser";
+import Stack from "../ui/Stack";
+import { Text } from "../ui/typography";
+import { StyleSheet } from "react-native";
+import { globalStyles } from "@src/styles/globalStyles";
+import Box from "../ui/Box";
```
Comment (whamester) [inline src/components/profile/StreakCard.tsx:6]: relative import

**Review by zeckkimon — APPROVED** [review]

**Pull request merged**